spec/controllers/concerns/verify_sp_attributes_concern_spec.rb

```diff
@@ -208,4 +184,107 @@
       end
     end
   end
+
+  describe '#reverified_after_consent?' do
```

Comment:
Could we enhance the specs for `needs_completion_screen_reason` to include one where we're expecting `:reverified_after_consent` as the reason?